Lib/idlelib/editor.py

```diff
@@ -188,6 +189,9 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
         vbar['command'] = text.yview
         vbar.pack(side=RIGHT, fill=Y)
         text['yscrollcommand'] = vbar.set
+        hbar['command'] = text.xview
+        hbar.pack(side='bottom', fill='x')
```

Comment:
I don't think this request is going to be merged, or it will require many more changes before it is merged, so I'm going to hold off on additional commits for now.